contentcuration/contentcuration/static/js/edit_channel/models.js

```diff
@@ -215,10 +215,13 @@ var ContentNodeCollection = BaseCollection.extend({
 			var assessmentCollection = new AssessmentItemCollection();
 			self.forEach(function(node){
 				node.get("files").forEach(function(file){
-					file.preset.id = file.preset.name ? file.preset.name : file.preset.id;
+					var to_add = new FileModel(file);
```

Comment:
Need to set id to include language id as Backbone collections don't allow for models with the same id. This causes problems for subtitles as all language variants have the base preset `video_subtitle`